<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk+" version="2.20"/>
  <!-- interface-naming-policy project-wide -->
  <object class="GtkListStore" id="liststore_sleep">
    <columns>
      <!-- column-name name -->
      <column type="gchararray"/>
      <!-- column-name value -->
      <column type="gint"/>
    </columns>
    <data>
      <row>
        <col id="0" translatable="yes">Suspend</col>
        <col id="1">1</col>
      </row>
      <row>
        <col id="0" translatable="yes">Hibernate</col>
        <col id="1">3</col>
      </row>
    </data>
  </object>
  <object class="GtkListStore" id="liststore_power">
    <columns>
      <!-- column-name name -->
      <column type="gchararray"/>
      <!-- column-name value -->
      <column type="gint"/>
    </columns>
    <data>
      <row>
        <col id="0" translatable="yes">Suspend</col>
        <col id="1">1</col>
      </row>
      <row>
        <col id="0" translatable="yes">Hibernate</col>
        <col id="1">3</col>
      </row>
      <row>
        <col id="0" translatable="yes">Do nothing</col>
        <col id="1">5</col>
      </row>
      <row>
        <col id="0" translatable="yes">Ask me</col>
        <col id="1">4</col>
      </row>
      <row>
        <col id="0" translatable="yes">Shutdown</col>
        <col id="1">2</col>
      </row>
    </data>
  </object>
  <object class="GtkListStore" id="liststore_critical">
    <columns>
      <!-- column-name name -->
      <column type="gchararray"/>
      <!-- column-name gint1 -->
      <column type="gint"/>
    </columns>
    <data>
      <row>
        <col id="0" translatable="yes">Hibernate</col>
        <col id="1">3</col>
      </row>
      <row>
        <col id="0" translatable="yes">Shutdown</col>
        <col id="1">2</col>
      </row>
    </data>
  </object>
  <object class="GtkSizeGroup" id="sizegroup_combos">
    <widgets>
      <widget name="combobox_critical"/>
      <widget name="combobox_button_power"/>
      <widget name="combobox_button_sleep"/>
      <widget name="combobox_sleep"/>
      <widget name="combobox_sleep_battery"/>
      <widget name="combobox_sleep_ac"/>
    </widgets>
  </object>
  <object class="GtkWindow" id="window_power">
    <property name="resizable">False</property>
    <child>
      <object class="GtkVBox" id="vbox_power">
        <property name="visible">True</property>
        <property name="border_width">12</property>
        <property name="spacing">12</property>
        <child>
          <object class="GtkVBox" id="vbox_ac_batt">
            <property name="visible">True</property>
            <child>
              <object class="GtkVBox" id="vbox_battery">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkHBox" id="hbox11">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkLabel" id="label6">
                        <property name="visible">True</property>
                        <property name="label" translatable="yes">Put the computer to sleep when on:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkVBox" id="vbox30">
                    <property name="visible">True</property>
                    <property name="spacing">6</property>
                    <child>
                      <object class="GtkHBox" id="hbox12">
                        <property name="visible">True</property>
                        <property name="spacing">6</property>
                        <child>
                          <object class="GtkHBox" id="hbox14">
                            <property name="visible">True</property>
                            <child>
                              <object class="GtkCheckButton" id="checkbutton_sleep_ac">
                                <property name="label" translatable="yes">AC power and inactive for:</property>
                                <property name="visible">True</property>
                                <property name="can_focus">True</property>
                                <property name="receives_default">False</property>
                                <property name="use_action_appearance">False</property>
                                <property name="draw_indicator">True</property>
                              </object>
                              <packing>
                                <property name="expand">False</property>
                                <property name="padding">12</property>
                                <property name="position">0</property>
                              </packing>
                            </child>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="position">0</property>
                          </packing>
                        </child>
                        <child>
                          <object class="GtkComboBoxText" id="combobox_sleep_ac">
                            <property name="visible">True</property>
                            <property name="model">liststore_time</property>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="fill">False</property>
                            <property name="position">1</property>
                          </packing>
                        </child>
                      </object>
                      <packing>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkHBox" id="hbox13">
                        <property name="visible">True</property>
                        <property name="spacing">6</property>
                        <child>
                          <object class="GtkHBox" id="hbox15">
                            <property name="visible">True</property>
                            <child>
                              <object class="GtkCheckButton" id="checkbutton_sleep_battery">
                                <property name="label" translatable="yes">Battery power and inactive for:</property>
                                <property name="visible">True</property>
                                <property name="can_focus">True</property>
                                <property name="receives_default">False</property>
                                <property name="use_action_appearance">False</property>
                                <property name="draw_indicator">True</property>
                              </object>
                              <packing>
                                <property name="expand">False</property>
                                <property name="padding">12</property>
                                <property name="position">0</property>
                              </packing>
                            </child>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="position">0</property>
                          </packing>
                        </child>
                        <child>
                          <object class="GtkComboBoxText" id="combobox_sleep_battery">
                            <property name="visible">True</property>
                            <property name="model">liststore_time</property>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="fill">False</property>
                            <property name="position">1</property>
                          </packing>
                        </child>
                      </object>
                      <packing>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkHBox" id="hbox_ac">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkHBox" id="hbox10">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkCheckButton" id="checkbutton_sleep">
                        <property name="label" translatable="yes">Put the computer to sleep when inactive for:</property>
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">False</property>
                        <property name="use_action_appearance">False</property>
                        <property name="draw_indicator">True</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBoxText" id="combobox_sleep">
                    <property name="visible">True</property>
                    <property name="model">liststore_time</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkVBox" id="vbox_actions">
            <property name="visible">True</property>
            <child>
              <object class="GtkHBox" id="hbox5">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkHBox" id="hbox6">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkLabel" id="label5">
                        <property name="visible">True</property>
                        <property name="label" translatable="yes">When power is critically low:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBoxText" id="combobox_critical">
                    <property name="visible">True</property>
                    <property name="model">liststore_critical</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkVBox" id="vbox_buttons">
            <property name="visible">True</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkHBox" id="hbox1">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkHBox" id="hbox7">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkLabel" id="label4">
                        <property name="visible">True</property>
                        <property name="label" translatable="yes">When the power button is pressed:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBoxText" id="combobox_button_power">
                    <property name="visible">True</property>
                    <property name="model">liststore_power</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkHBox" id="hbox4">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkHBox" id="hbox8">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkLabel" id="label3">
                        <property name="visible">True</property>
                        <property name="label" translatable="yes">When the sleep button is pressed:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBoxText" id="combobox_button_sleep">
                    <property name="visible">True</property>
                    <property name="model">liststore_sleep</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">2</property>
          </packing>
        </child>
        <child>
          <object class="GtkHBox" id="hbox_status">
            <property name="visible">True</property>
            <child>
              <object class="GtkHBox" id="hbox2">
                <property name="visible">True</property>
                <property name="border_width">12</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkImage" id="image_status">
                    <property name="visible">True</property>
                    <property name="stock">gtk-dialog-authentication</property>
                    <property name="icon-size">6</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkVBox" id="vbox4">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkHBox" id="hbox3">
                        <property name="visible">True</property>
                        <child>
                          <object class="GtkLabel" id="label_title">
                            <property name="visible">True</property>
                            <property name="label">Battery charging</property>
                            <attributes>
                              <attribute name="scale" value="1.500000"/>
                            </attributes>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="position">0</property>
                          </packing>
                        </child>
                      </object>
                      <packing>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label_description">
                        <property name="visible">True</property>
                        <property name="label">55 minutes until fully charged (6%)</property>
                      </object>
                      <packing>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">3</property>
          </packing>
        </child>
      </object>
    </child>
  </object>
  <object class="GtkSizeGroup" id="sizegroup_labels">
    <widgets>
      <widget name="hbox6"/>
      <widget name="hbox7"/>
      <widget name="hbox8"/>
      <widget name="hbox10"/>
      <widget name="hbox15"/>
      <widget name="hbox14"/>
    </widgets>
  </object>
  <object class="GtkListStore" id="liststore_time">
    <columns>
      <!-- column-name name -->
      <column type="gchararray"/>
      <!-- column-name value -->
      <column type="gint"/>
    </columns>
    <data>
      <row>
        <col id="0" translatable="yes">5 minutes</col>
        <col id="1">300</col>
      </row>
      <row>
        <col id="0" translatable="yes">10 minutes</col>
        <col id="1">600</col>
      </row>
      <row>
        <col id="0" translatable="yes">30 minutes</col>
        <col id="1">1800</col>
      </row>
      <row>
        <col id="0" translatable="yes">1 hour</col>
        <col id="1">3600</col>
      </row>
    </data>
  </object>
</interface>
